src/libbson/doc/bson_new_from_json.rst

```diff
@@ -21,8 +21,10 @@ Parameters
 Description
 -----------
 
-The ``bson_new_from_json()`` function allocates and initialize a new :symbol:`bson_t` by parsing the JSON found in ``data``. Only a single JSON object may exist in ``data`` or an error will be set and NULL returned.
-
+The ``bson_new_from_json()`` function allocates and initialize a new :symbol:`bson_t` by parsing the JSON found in ``data``.
+If there are multiple comma-separated JSONs in ``data``, the keys from all JSONs are merged in the returned BSON.
+If the first extra character after the last JSON is ``{``, all following characters are ignored and no error is set.
+Otherwise, an error will be set and NULL returned. 
 Errors
```

Comment:
Suggest adding tests for this behavior. That will prevent code changes from breaking this now documented behavior.